### ISSUE TITLE
Output bubble truncated to half of first line on multi-line outputs

### ISSUE THREAD

**Issue by weiji14** [nteract/hydrogen#898]
**Description:**

In the latest atom-beta release, multi-line outputs do not show up properly all the time as expected.

Expected:
![1bx7lquqll](https://user-images.githubusercontent.com/23487320/27666437-ce573b3c-5cc7-11e7-963c-0ee94bdf5ac0.gif)

Actual:
![oro8qvrpmi](https://user-images.githubusercontent.com/23487320/27666439-d93cc882-5cc7-11e7-9617-52f46230acff.gif)

**Steps to Reproduce:**

1. Type in this sample Python code into atom-beta editor build [1.19.0-beta2](https://github.com/atom/atom/releases/tag/v1.19.0-beta2).
```
    import random
    for i in range(15):
        print(random.random())
```
2. Set language to Python at lower right corner of atom-beta editor.
3. Press `Ctrl + Enter`

**Versions:**

Running version 1.18.0 of Hydrogen, and v 1.19.0-beta2 of Atom. I've found this problem on my Windows, and Linux (Debian and Ubuntu) workstations. Atom v 1.18 works fine per tests above.

**Logs:**
From my windows environment

```
utils.js:121 Hydrogen: findCodeBlock: 2
log @ utils.js:121
findCodeBlock @ code-manager.js:310
run @ main.js:463
hydrogenRun @ main.js:90
module.exports.CommandRegistry.handleCommandEvent @ <embedded>:21775
(anonymous) @ <embedded>:21513
module.exports.KeymapManager.dispatchCommandEvent @ <embedded>:66625
module.exports.KeymapManager.handleKeyboardEvent @ <embedded>:66416
module.exports.WindowEventHandler.handleDocumentKeyEvent @ <embedded>:18116
(anonymous) @ <embedded>:18019
utils.js:121 Hydrogen: getFoldRange: [2, 3]
log @ utils.js:121
getFoldRange @ code-manager.js:84
getFoldContents @ code-manager.js:89
findCodeBlock @ code-manager.js:320
run @ main.js:463
hydrogenRun @ main.js:90
module.exports.CommandRegistry.handleCommandEvent @ <embedded>:21775
(anonymous) @ <embedded>:21513
module.exports.KeymapManager.dispatchCommandEvent @ <embedded>:66625
module.exports.KeymapManager.handleKeyboardEvent @ <embedded>:66416
module.exports.WindowEventHandler.handleDocumentKeyEvent @ <embedded>:18116
(anonymous) @ <embedded>:18019
utils.js:121 Hydrogen: clearBubblesOnRow: 3
log @ utils.js:121
clearBubblesOnRow @ main.js:447
insertResultBubble @ main.js:360
_createResultBubble @ main.js:336
createResultBubble @ main.js:316
run @ main.js:473
hydrogenRun @ main.js:90
module.exports.CommandRegistry.handleCommandEvent @ <embedded>:21775
(anonymous) @ <embedded>:21513
module.exports.KeymapManager.dispatchCommandEvent @ <embedded>:66625
module.exports.KeymapManager.handleKeyboardEvent @ <embedded>:66416
module.exports.WindowEventHandler.handleDocumentKeyEvent @ <embedded>:18116
(anonymous) @ <embedded>:18019
utils.js:121 Hydrogen: clearBubblesOnRow: 3 ResultView {marker: DisplayMarker, element: div.hydrogen.marker, disposer: CompositeDisposable}
log @ utils.js:121
(anonymous) @ main.js:453
(anonymous) @ C:\Users\username\.atom\packages\Hydrogen\node_modules\lodash\lodash.js:4944
baseForOwn @ C:\Users\username\.atom\packages\Hydrogen\node_modules\lodash\lodash.js:3001
(anonymous) @ C:\Users\username\.atom\packages\Hydrogen\node_modules\lodash\lodash.js:4913
forEach @ C:\Users\username\.atom\packages\Hydrogen\node_modules\lodash\lodash.js:9359
clearBubblesOnRow @ main.js:448
insertResultBubble @ main.js:360
_createResultBubble @ main.js:336
createResultBubble @ main.js:316
run @ main.js:473
hydrogenRun @ main.js:90
module.exports.CommandRegistry.handleCommandEvent @ <embedded>:21775
(anonymous) @ <embedded>:21513
module.exports.KeymapManager.dispatchCommandEvent @ <embedded>:66625
module.exports.KeymapManager.handleKeyboardEvent @ <embedded>:66416
module.exports.WindowEventHandler.handleDocumentKeyEvent @ <embedded>:18116
(anonymous) @ <embedded>:18019
utils.js:121 Hydrogen: Kernel.execute: for i in range(15):
    print(random.random())
log @ utils.js:121
execute @ zmq-kernel.js:217
_createResultBubble @ main.js:343
createResultBubble @ main.js:316
run @ main.js:473
hydrogenRun @ main.js:90
module.exports.CommandRegistry.handleCommandEvent @ <embedded>:21775
(anonymous) @ <embedded>:21513
module.exports.KeymapManager.dispatchCommandEvent @ <embedded>:66625
module.exports.KeymapManager.handleKeyboardEvent @ <embedded>:66416
module.exports.WindowEventHandler.handleDocumentKeyEvent @ <embedded>:18116
(anonymous) @ <embedded>:18019
utils.js:121 Hydrogen: IO message: Message {idents: Array[1], header: Object, parent_header: Object, metadata: Object, content: Object…}
log @ utils.js:121
onIOMessage @ zmq-kernel.js:345
(anonymous) @ C:\Users\username\.atom\packages\Hydrogen\node_modules\jmp\index.js:347
emitMany @ events.js:127
emit @ events.js:201
Socket._emitMessage @ C:\Users\username\.atom\packages\Hydrogen\node_modules\zeromq\lib\index.js:632
Socket._flushRead @ C:\Users\username\.atom\packages\Hydrogen\node_modules\zeromq\lib\index.js:643
Socket._flushReads @ C:\Users\username\.atom\packages\Hydrogen\node_modules\zeromq\lib\index.js:679
_zmq.onReadReady @ C:\Users\username\.atom\packages\Hydrogen\node_modules\zeromq\lib\index.js:298
utils.js:121 Hydrogen: IO message: Message {idents: Array[1], header: Object, parent_header: Object, metadata: Object, content: Object…}
log @ utils.js:121
onIOMessage @ zmq-kernel.js:345
(anonymous) @ C:\Users\username\.atom\packages\Hydrogen\node_modules\jmp\index.js:347
emitMany @ events.js:127
emit @ events.js:201
Socket._emitMessage @ C:\Users\username\.atom\packages\Hydrogen\node_modules\zeromq\lib\index.js:632
Socket._flushRead @ C:\Users\username\.atom\packages\Hydrogen\node_modules\zeromq\lib\index.js:643
Socket._flushReads @ C:\Users\username\.atom\packages\Hydrogen\node_modules\zeromq\lib\index.js:679
_zmq.onReadReady @ C:\Users\username\.atom\packages\Hydrogen\node_modules\zeromq\lib\index.js:298
utils.js:121 Hydrogen: IO message: Message {idents: Array[1], header: Object, parent_header: Object, metadata: Object, content: Object…}
log @ utils.js:121
onIOMessage @ zmq-kernel.js:345
(anonymous) @ C:\Users\username\.atom\packages\Hydrogen\node_modules\jmp\index.js:347
emitMany @ events.js:127
emit @ events.js:201
Socket._emitMessage @ C:\Users\username\.atom\packages\Hydrogen\node_modules\zeromq\lib\index.js:632
Socket._flushRead @ C:\Users\username\.atom\packages\Hydrogen\node_modules\zeromq\lib\index.js:643
Socket._flushReads @ C:\Users\username\.atom\packages\Hydrogen\node_modules\zeromq\lib\index.js:679
_zmq.onReadReady @ C:\Users\username\.atom\packages\Hydrogen\node_modules\zeromq\lib\index.js:298
utils.js:121 Hydrogen: IO message: Message {idents: Array[1], header: Object, parent_header: Object, metadata: Object, content: Object…}
log @ utils.js:121
onIOMessage @ zmq-kernel.js:345
(anonymous) @ C:\Users\username\.atom\packages\Hydrogen\node_modules\jmp\index.js:347
emitMany @ events.js:127
emit @ events.js:201
Socket._emitMessage @ C:\Users\username\.atom\packages\Hydrogen\node_modules\zeromq\lib\index.js:632
Socket._flushRead @ C:\Users\username\.atom\packages\Hydrogen\node_modules\zeromq\lib\index.js:643
Socket._flushReads @ C:\Users\username\.atom\packages\Hydrogen\node_modules\zeromq\lib\index.js:679
_zmq.onReadReady @ C:\Users\username\.atom\packages\Hydrogen\node_modules\zeromq\lib\index.js:298
utils.js:121 Hydrogen: watchview running: 
log @ utils.js:121
(anonymous) @ watch.js:29
executeAction @ C:\Users\username\.atom\packages\Hydrogen\node_modules\mobx\lib\mobx.js:867
res @ C:\Users\username\.atom\packages\Hydrogen\node_modules\mobx\lib\mobx.js:858
(anonymous) @ watches.js:86
(anonymous) @ C:\Users\username\.atom\packages\Hydrogen\node_modules\mobx\lib\mobx.js:652
(anonymous) @ watches.js:86
executeAction @ C:\Users\username\.atom\packages\Hydrogen\node_modules\mobx\lib\mobx.js:867
res @ C:\Users\username\.atom\packages\Hydrogen\node_modules\mobx\lib\mobx.js:858
(anonymous) @ kernel.js:78
_callWatchCallbacks @ kernel.js:78
onIOMessage @ zmq-kernel.js:361
(anonymous) @ C:\Users\username\.atom\packages\Hydrogen\node_modules\jmp\index.js:347
emitMany @ events.js:127
emit @ events.js:201
Socket._emitMessage @ C:\Users\username\.atom\packages\Hydrogen\node_modules\zeromq\lib\index.js:632
Socket._flushRead @ C:\Users\username\.atom\packages\Hydrogen\node_modules\zeromq\lib\index.js:643
Socket._flushReads @ C:\Users\username\.atom\packages\Hydrogen\node_modules\zeromq\lib\index.js:679
_zmq.onReadReady @ C:\Users\username\.atom\packages\Hydrogen\node_modules\zeromq\lib\index.js:298
utils.js:121 Hydrogen: shell message: Message {idents: Array[0], header: Object, parent_header: Object, metadata: Object, content: Object…}
log @ utils.js:121
onShellMessage @ zmq-kernel.js:278
(anonymous) @ C:\Users\username\.atom\packages\Hydrogen\node_modules\jmp\index.js:347
emitMany @ events.js:127
emit @ events.js:201
Socket._emitMessage @ C:\Users\username\.atom\packages\Hydrogen\node_modules\zeromq\lib\index.js:632
Socket._flushRead @ C:\Users\username\.atom\packages\Hydrogen\node_modules\zeromq\lib\index.js:643
Socket._flushReads @ C:\Users\username\.atom\packages\Hydrogen\node_modules\zeromq\lib\index.js:679
_zmq.onReadReady @ C:\Users\username\.atom\packages\Hydrogen\node_modules\zeromq\lib\index.js:298


```

**Comment by t9md** [comment]
This is caused by the change made in Atom v1.19.0.

- Hydrogen use blockDecoration as output of code-evaluation(bubble).
- From Atom v1.19.0, blockDecoration's height become not being auto updated.
- To adjust the output area's height to it's content, each we need to manually **invalidate** previous blockDecoration measurement.
- There is #888 which is mainly as preparation.
- I believe fix for this issue will come next.

Related: atom/atom#14836

**Comment by lgeiger** [comment]
It may will be fixed upstream: https://github.com/atom/atom/pull/14844

**Comment by t9md** [comment]
Wow, understand, thanks. I didn't read updated comment in that thread.

**Comment by lgeiger** [comment]
A fix for this will very likely be included in Atom 1.19-beta3: https://github.com/atom/atom/pull/14929

**Comment by lgeiger** [comment]
This is fixed in Atom 1.19-beta3
